worker/modelcache/worker.go

```diff
@@ -324,3 +355,20 @@ func coreNetworkPortRanges(delta []multiwatcher.PortRange) []network.PortRange {
 	}
 	return ports
 }
+
+// lxdCharmProfiler massages a charm.Charm into a LXDProfiler inside of the
+// core package.
+type lxdCharmProfiler struct {
```

Comment:
This appears to be unused.

Reply:
removed, this area has changed around some.